### PR TITLE
[SDAN-485] Adding slugline to tooltip of coverages and coverage tiles…

### PR DESCRIPTION
**Pull request by mdhaman**
… in preview and fullscreen view.

### CODE REVIEW

File: assets/agenda/components/AgendaListItemIcons.jsx [hunk 1]
```diff
@@ -31,32 +29,45 @@ function AgendaListItemIcons({item, planningItem, group, hideCoverages, row}) {
...
+            return gettext('{{ type }} coverage {{slugline}} cancelled', {
+                type: getCoverageDisplayName(coverage.coverage_type),
+                slugline: slugline,
             });
         }
 
         if (coverage.workflow_status === WORKFLOW_STATUS.COMPLETED) {
-            return gettext('{{ type }} coverage available', {
-                type: getCoverageDisplayName(coverage.coverage_type)
+            return gettext('{{ type }} coverage{{ slugline }} available', {
```
Comment (petrjasek) [inline assets/agenda/components/AgendaListItemIcons.jsx]: missing space before `{{ slugline }}`

File: assets/agenda/components/AgendaListItemIcons.jsx [hunk 2]
```diff
@@ -31,32 +29,45 @@ function AgendaListItemIcons({item, planningItem, group, hideCoverages, row}) {
     });
 
     const getCoverageTootip = (coverage) => {
+        let slugline = coverage.item_slugline || coverage.slugline;
 
+        slugline = slugline ? ` '${slugline}'` : '';
         if (coverage.workflow_status === WORKFLOW_STATUS.DRAFT) {
-            return gettext('{{ type }} coverage {{ status }}', {
+            return gettext('Planned {{ type }} coverage{{ slugline }}', {
```
Comment (petrjasek) [inline assets/agenda/components/AgendaListItemIcons.jsx]: not sure now if space is missing in slugline or it's with this ` | ` in it?

**Comment by petrjasek** [comment]
I think we shouldn't add those miniconda binaries

File: assets/agenda/components/AgendaListItemIcons.jsx (hunk 1 above)
Comment (mdhaman) [reply assets/agenda/components/AgendaListItemIcons.jsx]: done

File: assets/agenda/components/AgendaListItemIcons.jsx (hunk 2 above)
Comment (mdhaman) [reply assets/agenda/components/AgendaListItemIcons.jsx]: it is with `|`.

**Comment by mdhaman** [comment]
@petrjasek Removed those unwanted files.

**Pull request merged**